src/json/index.js

```diff
@@ -0,0 +1,257 @@
+const data = {
```

Comment:
If we're going to add a stand-in JSON file, let's make it JSON, and not at .js file.

Reply:
In the overview component this changes to a JSON file